compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Scroll.kt

```diff
@@ -107,6 +107,14 @@ class ScrollState(initial: Int) : ScrollableState {
             }
         }
 
+    /**
+     * Size of the viewport on the scrollable axis, or 0 if still unknown.
+     */
+    // This property is internal until it is merged upstream, because we can't expose an API that
+    // doesn't exist on Android. Once it is merged upstream, it should be made public with an
+    // internal setter
+    internal var viewportSize: Int by mutableStateOf(0, structuralEqualityPolicy())
```

Comment:
Let's not merge this PR, until we discuss the new API with Google. In case if we have to provide this information somewhere else. This can affect our new API `v2.ScrollbarAdapter`/`v2.rememberScrollbarAdapter`

Reply:
Ok, let's talk in private how we can get Google's feedback on it.